src/main/java/com/khutircraftubackend/category/response/CategoryResponse.java

```diff
@@ -4,7 +4,6 @@
 
 @Builder
 public record CategoryResponse (
-        Long id,
```

Comment:
i think it's ok to keep id for category. 
especially if we return a collection of categories,
then having id field, client will be able to build a tree of categories.